client/my-sites/themes/theme-showcase.jsx

```diff
@@ -58,6 +60,98 @@ const optionShape = PropTypes.shape( {
 	action: PropTypes.func,
 } );
 
+const bannerShape = {
+	siteId: PropTypes.number,
+};
+
+const bannerClickHandler = ( themeId, tracksEvent, { siteId, getBannerUrl, getCurrentTheme } ) => {
+	const url = getBannerUrl( themeId, siteId );
+	const tracksData = {
+		site_id: siteId,
+		current_theme: getCurrentTheme( siteId ),
+		timestamp: new Date().toGMTString(),
+	};
+	recordTracksEvent( tracksEvent, tracksData );
+	page( url );
+};
+
+class PhotoBlogBanner extends PureComponent {
+	static propTypes = bannerShape;
+
+	render() {
+		const onClick = () => {
+			bannerClickHandler( 'photo-blog', 'calypso_showcase_banner_photo_blog_click', this.props );
```

Comment:
Instead of making the theme slug part of the event name, what about keeping the event name theme-agnostic (i.e. `calypso_showcase_banner_click`), and passing the theme slug as part of the event date?

(Also, maybe the event name should include `theme`, i.e. `calypso_theme_showcase_banner_click`.)

Reply:
This would also allow us to move the event tracking `onClick` handler to the underlying `ThemesBanner`, which makes more sense IMO.